devfile.yaml

```diff
@@ -31,7 +31,31 @@ commands:
       label: "[UD] build"
       component: tools
       workingDir: ${PROJECTS_ROOT}/che-dashboard
-      commandLine: "yarn build"
+      commandLine: "yarn --cwd packages/common build && yarn --cwd packages/dashboard-frontend build:dev && yarn --cwd packages/dashboard-backend build:dev"
```

Comment:
@dkwon17 I think it could be better to provide a separate command like ```yarn build:dev```

Reply:
Sounds good, fixed